exercises/rna-transcription/example/RNA.ceylon

```diff
@@ -0,0 +1,13 @@
+class Error() {}
+
+String|Error transcription(String dna) {
+  return dna.fold("" of String|Error)((String|Error r,
+                                       Character    c) =>   if(is Error r)
```

Comment:
I have no ground to stand on for this request other than "all other exercises do it this way", but I've had a space between `if` and the subsequent parenthesis.